appengine-jetty-managed-runtime/pom.xml

```diff
@@ -247,6 +243,18 @@
               </resources>
             </configuration>
           </execution>
+          <execution>
+            <phase>clean</phase>
+            <goals>
+              <goal>removeImage</goal>
+            </goals>
+            <configuration>
+              <imageName>appengine-mvn-opensource</imageName>
+              <imageTags>
+                <imageTag>latest</imageTag>
```

Comment:
This change no longer explicitly tags changes and as a result they automatically get tagged as "latest" but the plugin does not allow that for the removeImage goal - to remove "latest" the <imageTags> configuration needs to be specified. 

IOW, it does remove ones that were just built. If you have other tags for the image those will not be removed.